src/main/java/io/citrine/jcc/search/core/result/BaseMultiSearchResult.java

```diff
@@ -55,17 +55,30 @@ public Long getTook() {
     }
 
     /**
+     * Set the list of successes.  This overwrites any successes that are already saved.
+     * @param successes List of successes.
+     * @return This object.
+     */
+    @JsonSetter("successes")
```

Comment:
While that is _technically_ a bean, I meant something that would convey a little more meaning. Something that had methods like `getResult` and `getIsSuccess` so that the user didn't have to memorize what was at certain indices.